docs/spec_draft.md

```diff
@@ -385,6 +385,86 @@ operation.
   // %z: [[false, true], [true, true]]
 ```
 
+## stablehlo.rsqrt
+
+`stablehlo.rsqrt(operand) -> result`
+
+### Semantics
+
+Performs element-wise rsqrt operation on `operand` tensor and produces a `result` tensor.
+$$rsqrt(x) = \frac{1}{\sqrt{x}}$$
+
+### Attributes
+
+| Name | Type |
+|-|-|
+| `value` | `tensor` of floating-point or complex element types |
+
+### Operands
+
+| Name | Type |
+|-|-|
+| `value` | constant n-dimensional data of floating-point, or complex element types |
+
+### Results
+
+| Name | Type |
+|-|-|
+| `result` | `tensor` of floating-point, or complex element types |
+
+### Constraints
+
+  * Supported shapes: all static shapes.
+  * `result` must have the same type as that of `operand`.
+
+### Examples
+
+```mlir
+// %operand: [[1.0, 4.0], [9.0, 25.0]]
+%result = stablehlo.rsqrt %operand : tensor<2x2xf32>
+// %result: [[1.0, 0.5], [0.33333343, 0.2]]
+```
+
+## stablehlo.sqrt
+
+`stablehlo.sqrt(operand) -> result`
+
+### Semantics
```

Comment:
Also can we say something about negation int values?

Reply:
Or just point out the appropriate operation from IEEE 754 standard that this op will be implementing. That way we can skip the details.

Reply:
Done